src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -267,6 +279,59 @@ static uint8_t readLevelControlCurrentLevel(EndpointId endpoint)
     return currentLevel;
 }
 
+static uint8_t readColorLoopActiveAttribute(EndpointId endpoint)
+{
+    uint8_t isColorControleActive;
+    EmberAfStatus status;
+
+    status = emberAfReadServerAttribute(endpoint, ZCL_COLOR_CONTROL_CLUSTER_ID, ZCL_COLOR_CONTROL_COLOR_LOOP_ACTIVE_ATTRIBUTE_ID,
```

Comment:
Please use Accessors.h for all attribute reads/writes.

Reply:
can we initialize isColorControleActive to false and skip the `status != EMBER_ZCL_STATUS_SUCCESS`